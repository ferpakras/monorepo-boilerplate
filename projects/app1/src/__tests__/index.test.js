import React from "react";
import "@testing-library/jest-dom/extend-expect";
import { render } from "@testing-library/react";
import PageExample from "../pages";

test("test", () => {
  const { container } = render(<PageExample />);

  expect(container).toMatchSnapshot();
});
