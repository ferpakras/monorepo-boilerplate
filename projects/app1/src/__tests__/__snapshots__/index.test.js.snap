// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test 1`] = `
<div>
  <h1>
    ALOOOU
  </h1>
</div>
`;
